docs/configuration/other.md

```diff
@@ -84,3 +85,7 @@ The `app_grpc` section configures the App gRPC API server. This server is used f
 | `cors`         | Boolean | Dictates if CORS is active for the app-specific gRPC API server. Default: `true`.                                                       |
 | `web`          | Boolean | Activates HTTP/1 + web support for the app-specific gRPC API server. Default: `true`.                                                   |
 | `enabled`      | Boolean | Indicator to enable or disable the app-specific gRPC API server. Default: `true`.                                                       |
+
+### Default Query Limit
+
+The `default_max_records_num` is an   `Integer` parameter that specifies the maximum number of records returned from queries. The default value is 50.
```

Comment:
"maximum" -> "default limit of"

Too many spaces before `Ingeter`?

Reply:
addressed.